bindata/pdb/console-pdb.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: console-pdb
```

We dont need to call instance after the resource, should just contain the name of the component
```suggestion
  name: console
```